platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```diff
@@ -1718,13 +1724,10 @@
 									}
 									.SendKudosButtonTemplate {
 										.kudosLabel {
-											color: @baseColorLight;
+											color: @tertiaryColor;
```

Comment:
the color should be primary instead

Reply:
@boubaker I think that it should be good since this class is only used for sending a kudo from a comment